docs/adr/0008-license-attribution.md

```diff
@@ -0,0 +1,35 @@
+# 8. License Attribution
+
+Date: 2021-08-23
+
+## Status
+
+historical
+
+## Context
+
+Stencil is a distributable that contains third party packages that helps it compile projects. Examples include the 
+TypeScript compiler, rollup, etc. Those projects need to be properly attributed per their respective licenses.
+
+## Options
+
+N/A - the decision described this ADR predates the current Stencil team.
+
+## Decision
+
+Third party scripts that are included in the Stencil output shall be placed in the `entryDeps` field in a 
```

Comment:
This is all great to know, thank you for spelling it out so plainly. Up until now, I had no idea about this.